src/panels/lovelace/editor/hui-dialog-save-config.ts

```diff
@@ -19,13 +19,15 @@ import "../../../components/ha-formfield";
 import "../../../components/ha-svg-icon";
 import "../../../components/ha-switch";
 import "../../../components/ha-yaml-editor";
+import type { LovelaceConfig } from "../../../data/lovelace";
 import type { HassDialog } from "../../../dialogs/make-dialog-manager";
 import { haStyleDialog } from "../../../resources/styles";
 import type { HomeAssistant } from "../../../types";
 import { documentationUrl } from "../../../util/documentation-url";
+import { expandLovelaceConfigStrategies } from "../strategies/get-strategy";
 import type { SaveDialogParams } from "./show-save-config-dialog";
 
-const EMPTY_CONFIG = { views: [] };
+const EMPTY_CONFIG: LovelaceConfig = { views: [{ title: "Home" }] };
```

Comment:
This one has bothered me for so long. When it's emtpy, it's not clear how you can add a card, because we have no view. Just adding a stub view to get creation of cards going.